core/store/migrate/migrate.go

```diff
@@ -0,0 +1,82 @@
+package migrate
```

Comment:
The bulk of changes are in this file, migrations just got adapted to fit the new format without any changes.